west.yml

```diff
@@ -59,7 +59,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: v3.3.99-ncs1
+      revision: pull/1197/head
```

Comment:
`6fb229ca3c88b3c7c1ec9b0a2995703032eae4a8`

Reply:
updated and rebased